README.md

```diff
@@ -25,7 +25,7 @@ Open http://localhost:8765 to view it in the browser.
    docker run --rm -ti --name ydb-local -h localhost \
       -p 8765:8765 \
       -e MON_PORT=8765 \
-      cr.yandex/yc/yandex-docker-local-ydb:latest
+      ghcr.io/ydb-platform/local-ydb:nightly
```

Comment:
It looks like images from `ghcr.io/ydb-platform/local-ydb` should always be used instead of `cr.yandex/yc/yandex-docker-local-ydb`

https://github.com/ydb-platform/ydb/pkgs/container/local-ydb/122692891?tag=latest
https://github.com/ydb-platform/ydb/pkgs/container/local-ydb/236866741?tag=24.1

Can you also change images in the following doc places:
1) **Preview** - use latest tag
2) **YDB docker images** - change the description, removing old tags and mentioning new ones
3) **CI** - use nightly tag
4) **Making a production bundle** - use latest tag